Dockerfile

```diff
@@ -62,6 +62,8 @@ RUN --mount=type=ssh,uid=1000 sed -e s/return//g -i ~/.bashrc && \
     source ~/.bashrc && \
     # Set all git repositories to use HTTPS urls (Needed for local image builds)
     tue-env config ros-"$TUE_ROS_DISTRO" git-use-https && \
+    # Update branch of targets repository
+    tue-get update --branch="$BRANCH" && \
```

Comment:
Why not add the branch option to `tue-get install ros`? Also the branch option should be empty in case the branch is equal to the default branch.

Reply:
The branch option is equal to empty by default and the script build-docker-image passes an empty string in the default case. 

True, I could essentially just pass that arg to `tue-get install ros`, don't know what I was thinking last night :)